packages/cli/src/controller/codegen-controller.ts

```diff
@@ -60,11 +69,23 @@ export function processFields(className: string, fields: GraphQLFieldMap<unknown
   return fieldList;
 }
 
-// 1. Loop all entities and render it
-export function generateSchema(): void {
-  const schema = buildSchema('./schema.graphql');
-  const extractEntities = getAllEntities(schema);
+//1. Prepare models directory and load schema
+export async function codegen(projectPath: string): Promise<void> {
+  projectDir = projectPath;
+  try {
+    await promisify(rimraf)(path.join(projectDir, '/src/types'));
```

Comment:
don't add the leading `/`, in case projectDir is null and you might delete some important files by accident